### PR TITLE
Feature/api 15 command set

### PR DESCRIPTION
**Pull request by renemarc**
Solution for #9

### CODE REVIEW

File: lightpack.py [hunk 1]
```diff
@@ -494,12 +554,14 @@ def fail(cause = None):
 			fail(e)
 
 		# Check greeting and reported API version
-		match = re.match(r'^Lightpack API v(\S+)', greeting)
-		version = StrictVersion(match.group(1))
-		if version < API_VERSION_GTE or version > API_VERSION_LTE:
-			fail("API version (%s) is not supported" % version)
-		if not match:
-			print(match)
+		match = re.findall(r'API v?(\d+(?:\.\d+)?)', greeting)
```
Comment (tremby) [inline lightpack.py:537]: I don't like that the "start of string" anchor was removed here. Was that necessary?

File: lightpack.py [hunk 2]
```diff
@@ -763,3 +824,12 @@ def __init__(self, command, response, expected):
 		self.command = command
 		self.response = response
 		self.expected = expected
+class CommandNotSupportedError(RuntimeError):
```
Comment (tremby) [inline lightpack.py:809]: Should be a blank line between classes.

**Review by tremby — CHANGES_REQUESTED** [review]
Mostly looks great, just one query and one code style thing to fix.

File: lightpack.py (hunk 1 above)
Comment (renemarc) [reply lightpack.py:537]: The Prismatik fork appends its version after Lightpacks' original string, like so: `Lightpack API v1.4 - Prismatik API v2.2 (type "help" for more info)`. This updated regex is indeed looser on the position, but to compensate is stricter on the version matching.

File: lightpack.py (hunk 2 above)
Comment (renemarc) [reply lightpack.py:809]: Agreed. Should we add blank lines for the other classes here too, or in another PEP8 PR?

**Comment by tremby** [comment]
Ah, I didn't realize they were missing elsewhere. Yes, let's comply with
PEP8 across the board in a separate commit after all this.

I'll have some time later to have another look at everything.

File: lightpack.py (hunk 1 above)
Comment (tremby) [reply lightpack.py:537]: OK.

File: lightpack.py (hunk 2 above)
Comment (tremby) [reply lightpack.py:809]: (Ah replying by email apparently didn't put the comment in the right place. Moving it to here...)

Ah, I didn't realize they were missing elsewhere. Yes, let's comply with PEP8 across the board in a separate commit after all this. But please at least add blank lines before the new classes you're adding.

Comment (renemarc) [reply lightpack.py:809]: Fixed!

**Comment by renemarc** [comment]
PR's branch rebased on **feature/api-14-command-set** to avoid conflicts when eventually merging.

**Comment by tremby** [comment]
I've manually rebased and merged these changes.

**Pull request closed without merging**